src/test/java/io/github/murdos/easyrandom/protobuf/Protobuf2MessageGenerationTest.java

```diff
@@ -18,6 +18,8 @@
 import com.google.protobuf.StringValue;
 import io.github.murdos.easyrandom.protobuf.testing.proto2.Proto2Enum;
 import io.github.murdos.easyrandom.protobuf.testing.proto2.Proto2Message;
+import io.github.murdos.easyrandom.protobuf.testing.proto3.Proto3Enum;
+import io.github.murdos.easyrandom.protobuf.testing.proto3.Proto3Message;
```

Comment:
I think it's better to keep Proto2 and Proto3 tests data separate and not mix them: Protobuf2MessageGenerationTest should only use proto2 messages.
```suggestion
```